### ISSUE TITLE
UMD export

### ISSUE THREAD

**Issue by marcobiedermann** [osano/cookieconsent#491]
Could you please add the option for UMD exports?
This would allow you to install the package via npm and `import` / `require` it in your application. Right now it is just attached to the global `window` object

Current behavior:
```js
import 'cookieconsent';

window.cookieconsent.initialise({ … });
```

Expected behavior:
```js
import cookieconsent from 'cookieconsent';

cookieconsent.initialise({ … });
```

This would require to export the package if UMD is supported

Fix:
```js
if (typeof exports !== 'undefined') {
  module.exports = cookieconsent;
} else {
  window.cookieconsent = cookieconsent;
}
```

Or even better, use the default ES2015 `export` syntax

```js
export default cookieconsent;
```

**Comment by elliottregan** [comment]
@alexmorleyfinch @piiupilt Would you except a PR to support UMD or ES Modules?

**Comment by relicmelex** [comment]
Resolved with `e3fc715`
